images/update_kcp_stable/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
+LABEL build-date= \
+      com.redhat.build-host= \
+      description="This image provides binaries and a script to update image tags on Gitlab repo" \
```

Comment:
Do reference gitlab specifically. The description should be generic as this image might handle GitLab or other SCM in the future. E.g. `This image provides the required tooling to update the image tags on the SRE gitops repository`.
Also impact `io.k8s.description` and `summary`.